test/integration/resultsPage.js

```diff
@@ -15,136 +15,216 @@ const resultsRoute = `${constants.SITE_ROOT}/results/`;
-
-        const $ = cheerio.load(res.text);
-
-        const resultsHeader = $('.results__header').text();
-        expect(resultsHeader).to.contain(`GP surgery matching '${search}'`);
-
-        const searchResults = $('.results__item--nearby');
-        expect(searchResults.length).to.equal(1);
-
-        expect($('.link-back:first-of-type').eq(0).text()).to.equal('Back');
-        expect($('.link-back:first-of-type').eq(1).text()).to.equal('Back');
-        expect($('.link-back').attr('href')).to.equal(`${constants.SITE_ROOT}`);
-        done();
-      });
-  });
-
-  it('should return an object containing GP surgeries matching the query', (done) => {
-    const search = 'David';
-    chai.request(app)
-      .get(resultsRoute)
-      .query({ search })
-      .end((err, res) => {
-        iExpect.htmlWith200Status(err, res);
-
-        const $ = cheerio.load(res.text);
+  describe('page layout', () => {
+    it('should contain HTML', (done) => {
+      const search = 'Surgery';
+      chai.request(app)
+        .get(resultsRoute)
+        .query({ search })
+        .end((err, res) => {
+          iExpect.htmlWith200Status(err, res);
+          done();
+        });
+    });
+    it('should contain a back link', (done) => {
+      const search = 'Surgery';
+      chai.request(app)
+        .get(resultsRoute)
+        .query({ search })
+        .end((err, res) => {
+          const $ = cheerio.load(res.text);
 
-        const resultsHeader = $('.results__header').text();
-        expect(resultsHeader).to.contain(`GP surgeries matching '${search}'`);
+          expect($('.link-back:first-of-type').eq(0).text()).to.equal('Back');
+          expect($('.link-back:first-of-type').eq(1).text()).to.equal('Back');
+          expect($('.link-back').attr('href')).to.equal(`${constants.SITE_ROOT}`);
+          done();
+        });
+    });
+    it('should contain a a header with the search string', (done) => {
+      const search = 'Surgery';
+      chai.request(app)
+        .get(resultsRoute)
+        .query({ search })
+        .end((err, res) => {
+          const $ = cheerio.load(res.text);
 
-        done();
+          const resultsHeader = $('.results__header').text();
+          expect(resultsHeader).to.contain(`GP surgeries matching '${search}'`);
+          done();
+        });
+    });
+    describe('matching surgeries found', () => {
+      describe('single match', () => {
+        const search = 'Idle';
+        it('should have only one result', (done) => {
+          chai.request(app)
+            .get(resultsRoute)
+            .query({ search })
+            .end((err, res) => {
+              const $ = cheerio.load(res.text);
+
+              const searchResults = $('.results__item--nearby');
+              expect(searchResults.length).to.equal(1);
+
+              done();
+            });
+        });
+        it('should display singular message text', (done) => {
+          chai.request(app)
```

Comment:
Can this request and the one above be incorporated into one and moved into the `describe` block so the validation of the contents of cheerio can be tested within the `it` blocks. This should reduce duplication and ease readability.